web/sw.js

```diff
@@ -0,0 +1,17 @@
+const CACHE_NAME = 'nw-cache';
+const CACHED_ROUTES = [
+  '/',
+];
+
+this.addEventListener('install', (e) => {
+  e.waitUntil(
+    caches.open(CACHE_NAME).then(cache => cache.addAll(CACHED_ROUTES))
```

Comment:
This seems extremely aggressive and I'm not sure it'll ever invalidate the cache. Since sw.js is static I don't think the service worker will ever be updated.

Reply:
Ah you're right, seems that it stops on the installing stage. I think if I piped some dummy statement (`var a = random_string`) to the end of `sw.js` in our build process it would fix the issue. But using a plugin does seem a lot cleaner. :thinking: